package.json

```diff
@@ -16,7 +16,7 @@
     "patch:hosts": "fec patch-etc-hosts",
     "start": "HOT=true fec dev",
     "test": "jest",
-    "postinstall": "rimraf .cache",
+    "postinstall": "ts-patch install && rimraf .cache",
```

Comment:
do we need this? I assumed it was added with the new frontends-config

Reply:
Yep, I will remove it until we will update the package :+1: